### ISSUE TITLE
Cemetech master index

### ISSUE THREAD

**Issue by TC01** [TC01/calcpkg#6]
Okay, okay, so, we have working Cemetech support... but it's done by spidering the download category pages and not by reading in an index file, which would obviously be ideal.

I'm going to leave this one open until Cemetech has a master index file and we can just use that.

**Comment by Eeems** [comment]
Have you reached out to Kerm about this yet?

**Comment by TC01** [comment]
Yes, a few times. He's said he's willing to do implement a master index file, but has been pretty busy and is having trouble finding time to do it. (Which I can sympathize with).
